### PR TITLE
Upgrade dkan to 1.13

### PR DESCRIPTION
**Pull request by dkinzer**

### CODE REVIEW

File: config/settings.custom.php [hunk 1]
```diff
@@ -50,6 +50,9 @@ function devinci_custom_environment_switch($target_env, $current_env) {
       //db_query("DELETE FROM search_api_index where server IS NULL");
       db_query("UPDATE search_api_server set enabled = 0 WHERE machine_name <> 'local_solr_server'");
 
+      //Disable clamav
+      variable_set('clamav_enabled', FALSE);
```
Comment (dkinzer) [inline config/settings.custom.php]: We already control wether clamav is enabled or disabled via config.yml. Also changes in this file do not get propogated to sites when doing updates (nothing in the config folder does)

Comment (jacintocapote) [inline config/settings.custom.php]: But we are disabling for local environments. Is fine this process? Or we need to do via config.yml too?

Comment (dharizza) [inline config/settings.custom.php]: We should do it via config.yml; and anyways, we shouldn't touch things inside config/ dir in dkan_starter, so I've just removed that. If we need to do something like that for any client site, we should add it in their specific settings.custom.php.

**Comment by acouch** [comment]
Tests pass!


giphy

**Pull request merged**